activation/interface.go

```diff
@@ -58,7 +58,12 @@ type postSetupProvider interface {
 	Status() *PostSetupStatus
 	ComputeProviders() []PostSetupComputeProvider
 	Benchmark(p PostSetupComputeProvider) (int, error)
-	StartSession(context context.Context, opts PostSetupOpts) error
+	// InitializerConfig builds the config to be used by initialization in calls to
+	// StartSession. Having this function be separate from StartSession provides a
+	// means to understand if the post configuration is valid before kicking off a
+	// very long running task (StartSession can take hours to complete).
```

Comment:
These interfaces are private (and primarily exist to be mocked in tests). No need to document the method again if the actual implementation already has a godoc 🙂

Reply:
Right, thanks :+1: